Composer/packages/lib/code-editor/src/lg/LgSpeechModalityToolbar.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { FluentTheme } from '@uifabric/fluent-theme';
+import formatMessage from 'format-message';
+import {
+  ContextualMenuItemType,
+  IContextualMenuItemProps,
+  IContextualMenuItemRenderFunctions,
+} from 'office-ui-fabric-react/lib/ContextualMenu';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import * as React from 'react';
+
+import { ItemWithTooltip } from '../components/ItemWithTooltip';
+
+import { jsLgToolbarMenuClassName } from './constants';
+import { LgEditorToolbar, LgEditorToolbarProps } from './LgEditorToolbar';
+
+const menuItemStyles = {
+  fontSize: FluentTheme.fonts.small.fontSize,
+};
+
+const ssmlHeaderHelpUrl =
+  'https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/speech-synthesis-markup';
+const ssmlHeaderTooltipProps = { calloutProps: { layerProps: { className: jsLgToolbarMenuClassName } } };
+
+export type SSMLTagType = 'break' | 'prosody' | 'audio';
+
+type Props = Omit<LgEditorToolbarProps, 'moreMenuItems'> & {
+  onInsertSSMLTag: (tagType: SSMLTagType) => void;
+};
+
+export const LgSpeechModalityToolbar = React.memo((props: Props) => {
+  const { onInsertSSMLTag, ...restProps } = props;
+
+  const renderHeaderContent = React.useCallback(
+    (itemProps: IContextualMenuItemProps, defaultRenders: IContextualMenuItemRenderFunctions) => (
+      <ItemWithTooltip
+        itemText={defaultRenders.renderItemName(itemProps)}
+        tooltipId="ssml-menu-header"
+        tooltipProps={ssmlHeaderTooltipProps}
+        tooltipText={formatMessage.rich('To learn more about SSML Tags, <a>visit this document</a>.', {
+          a: ({ children }) => (
+            <Link key="ssml-menu-header-link" href={ssmlHeaderHelpUrl} target="_blank">
+              {children}
+            </Link>
+          ),
+        })}
+      />
+    ),
+    []
```

Comment:
useCallback with empty array of dependencies.  Intentional?

Reply:
it renders a static header given the args, no deps